swap_meet/vendor.py

```diff
@@ -1,2 +1,62 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
+    def __init__(self, inventory = None):
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+    
+    def add(self, item):
+        result = item
+        self.inventory.append(result)
+        return result
+    
+    def remove(self, item):
+        result = item
+        if item in self.inventory:
+            self.inventory.remove(result)
+        else:
+            return False
+        return result
```

Comment:
Consider first testing for the edge case, and then moving the main functionality of your function outside the `if/else`. This programming pattern is called a guard clause and can among other things enhance the readability of your code.

```suggestion
        if item not in self.inventory:
             return False
        self.inventory.remove(result)
        return item
```

In addition, assigning `result = item` means we need to take a step to understand what result is. Consider simply returning `item`